modules/firelens_log_config/secrets.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_ssm_parameter" "datadog_key" {
+  name   = "/${var.ssm_prefix}/${var.environment_name}/DATADOG_KEY"
+  type   = "SecureString"
+  value  = "ChangeMeInTheWebConsole"
+  key_id = aws_kms_key.for_ssm_params.key_id
+  overwrite = true
+
+  tags = var.tags
+
+  lifecycle {
+    ignore_changes = [value]
+  }
+}
+
+resource "aws_kms_key" "for_ssm_params" {
```

Comment:
We have a kms key for the app we could use for this already in https://github.com/smartrent/nerves_hub_infrastructure/blob/ddd6c2c087d2b8d48f26fb28a242027f0e28972a/all_environments/secrets.tf#L1

I think when this was set up though we might not have set up the other secrets to use that key. That was not intentional iirc. 🤷‍♀️

Reply:
Ah so you're thinking have the kms key be a variable so we just have the one in the deployment?

Reply:
This now does this ✅

Reply:
See the [draft PR linked in the updated description or this link](https://github.com/smartrent/nerves_hub_infrastructure/pull/68)